v7/base-austere/templates/feed_helper.tmpl

```diff
@@ -0,0 +1,61 @@
+## -*- coding: utf-8 -*-
+<%def name="head(name=None)">
+    <%
+        if name is None:
+            atom_ref = 'index_atom'
+            rss_ref = 'rss'
+            label = ''
+        else:
+            atom_ref = kind + '_atom'
+            rss_ref = kind + '_rss'
+            label = ' for ' + kind + ' ' + name
```

Comment:
Hm. I’d really like to see something like this upstream, but with i18n support, optional JSONFeed support, and without a raw Python block (which are not allowed in Jinja2) — changes to the core codebase to allow this everywhere would be more welcome.